src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private double side;
+
+    public Square() {
+    }
+
+    public Square(double side) {
+        this.typeOfFigure = TypeOfFigure.SQUARE;
+        this.area = getQuadrangularArea(side, side);
+        this.side = side;
```

Comment:
Creating a figure, don't pass expressions in the constructor. Create separate variables and pass them on for better code readability. Also, consider passing the color as a parameter to your figure constructors and setting it there, instead of leaving it uninitialized.